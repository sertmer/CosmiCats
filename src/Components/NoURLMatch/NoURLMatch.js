import React from 'react';
import { Link } from 'react-router-dom'
import './NoURLMatch.scss'

export const NoURLMatch = () => {
  return (
    <div className='no-match-div'>
      <h2 className='no-match-h2'>No matching url was found</h2>
      <Link to='/'>
        <button type='button' className='no-match-btn'>click here to go home</button>
      </Link>
    </div>
  )
}

export default NoURLMatch;