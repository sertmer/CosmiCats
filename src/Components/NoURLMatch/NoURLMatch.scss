@import '../../_scss/variables';
@import '../../_scss/mixins';

.no-match-div {
  display: flex;
  flex-direction: column;
  width: 100%;
  text-align: center;
  font-size: xx-large;
}

.no-match-btn {
  text-decoration: none;
  width: 20%;
  @include rounded-corners(5px);
  background: rgb(69,198,141);
  background: linear-gradient(
    0deg, rgba(69,198,141,1) 0%, 
    rgba(73,210,149,1) 35%, 
    rgba(85,246,175,1) 100%
  );
  padding: 3%;
  @include rounded-corners(5px);
  font-size: 1em;
  font-family: $main-font;
  -webkit-transform: perspective(1px) translateZ(0);
  transform: perspective(1px) translateZ(0);
  -webkit-transition-duration: 0.4s;
  transition-duration: 0.4s;
  -webkit-transition-property: transform;
  transition-property: transform;
}

.no-match-h2 {
  color: $alien-green;
}