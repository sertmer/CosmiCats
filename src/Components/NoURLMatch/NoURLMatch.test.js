import React from 'react';
import { NoURLMatch } from './NoURLMatch';
import { shallow } from 'enzyme';

describe('NoURLMatch', () => {
  let wrapper

  describe('NoURLMatch unit tests', () => {
    it('should match the snapshot', () => {
      wrapper = shallow(<NoURLMatch />)
      expect(wrapper).toMatchSnapshot()
    })
  })
  
})