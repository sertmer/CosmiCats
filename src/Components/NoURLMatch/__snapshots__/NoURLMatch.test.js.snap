// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoURLMatch NoURLMatch unit tests should match the snapshot 1`] = `
<div
  className="no-match-div"
>
  <h2
    className="no-match-h2"
  >
    No matching url was found
  </h2>
  <Link
    to="/"
  >
    <button
      className="no-match-btn"
      type="button"
    >
      click here to go home
    </button>
  </Link>
</div>
`;
